genai_cookbook/nbs/1-introduction-to-rag.md

```diff
@@ -1,6 +1,12 @@
-# RAG overview
+# Agents overview
```

Comment:
Wonder if we should rename this file to `introduction-to-agents.md` and if we can redirect the original file (or keep it but just manually include a link to the new page from there)